virtool/app.py

```diff
@@ -21,6 +21,7 @@
     shutdown_executors,
     shutdown_redis,
     shutdown_scheduler,
+    shutdown_auth_client,
```

Comment:
Need to be specific as this can be confused with authentication.
```suggestion
    shutdown_authorization_client,
```